### PR TITLE
Add hashboot.md under security

### PR DESCRIPTION
**Pull request by teldra**

### CODE REVIEW

**Comment by pltrz** [comment]
Use `vmdfmt` to format. See [here](https://docs.voidlinux.org/contributing/void-docs/style-guide.html#formatting).

**Comment by teldra** [comment]
Ah, thx. Should be corrected.

File: src/config/security/hashboot.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+"internal:laptop=force_I_want_a_brick"), the following must be set in
+`/etc/hashboot.cfg`:
+
+```
+PROGRAMMER="internal:laptop=force_I_want_a_brick"
+```
+
+# Usage
+
+- Run "hashboot index" to generate checksums and a backup for /boot and MBR
```
Comment (bobertlo) [inline src/config/security/hashboot.md]: `hashboot index` and the same for the next two lines

File: src/config/security/hashboot.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+```
+
+to compose the configurationfile and generate the index of the chosen options.
+
+> If this is not run after installation, next boot will stop with an emergency
+> shell.
+
+## Options
+
+The options are chosen by a bitmask:
```
Comment (bobertlo) [inline src/config/security/hashboot.md]: How are these options passed to hashboot?

Comment (teldra) [reply src/config/security/hashboot.md]: Should be clear now.

File: src/config/security/hashboot.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+to compose the configurationfile and generate the index of the chosen options.
+
+> If this is not run after installation, next boot will stop with an emergency
+> shell.
+
+## Options
+
+The options are chosen by a bitmask and set interactively at first run or in
+`etc/hashboot.cfg`:
```
Comment (bobertlo) [inline src/config/security/hashboot.md]: Not to nitpick but maybe it would be clearer as:
```
# 001=mbr, 010=files, 100=bios
CKMODES=101
```

Comment (teldra) [reply src/config/security/hashboot.md]: Restructured. I hope, its better now but open for changes.

**Comment by teldra** [comment]
Should I change anything else?

**Comment by bobertlo** [comment]
@teldra sorry I was very busy. I didn't realize noone else was working on these ;)

I hate to ask for more changes, this is just a basic formatting matter.

**Comment by teldra** [comment]
@bobertlo no problem! maybe now its better?

**Pull request merged**